maintenance/idr0021-data-prep.md

```diff
@@ -0,0 +1,90 @@
+
+Workshop data preparation (idr0021)
+===================================
+
+This document details the steps to prepare data from IDR for a workshop demonstrating
+analysis with Fiji, usage of Map Annotations and OMERO.tables and filtering with OMERO.parade.
+We use IDR0021, which is a Project containing 10 Datasets with a total of ~400 Images.
+
+
+Download IDR data
+=================
+
+External users
+--------------
+
+Please contact the OME team for access to download IDR data.
+
+Internal users
+--------------
+
+Follow instructions on the
+[IDR submission workflow](https://docs.google.com/document/d/1TmBZ43_yhiO3AOua8oMk4mPWKWJtpeYNc2KLP17h-1I/edit#) to download the data via ```ssh``` from ```idr0-slot3```.
+
+
+Prepare IDR-metadata and import
+===============================
+
+Clone https://github.com/IDR/idr-metadata and edit
+```idr-metadata/idr0021-lawo-pericentriolarmaterial/experimentA/idr0021-experimentA-filePaths.tsv```
+to point ALL paths at the location of the data downloaded above.
+e.g.
+```Dataset:name:CDK5RAP2-C	/full/path/to/data/idr0021/CDK5RAP2-C/Centrin_PCNT_Cep215_20110506/Centrin_PCNT_Cep215_20110506_Fri-1545_0_SIR_PRJ.dv```
+
+
+If you don't want to use in-place import, comment out this line in idr-metadata/bulk.yml:
+
+	transfer: "ln_s"
+
+
+Do the bulk import:
+
+	$ cd idr-metadata/idr0021-lawo-pericentriolarmaterial/experimentA/
+	$ omero import --bulk idr0021-experimentA-bulk.yml
+
+
+In the webclient, create a Project 'idr0021' and add the 10 new Datasets created above.
+
+
+Add Map Annotations from IDR
+============================
+
+Edit the ```maintenance/idr_get_map_annotations.py``` with the ID of the 'idr0021' Project created
```

Comment:
As the step before retrieves the annotations client side, the alternative here could have been to create the annotation using the IDR/populate metadata workflow:

```
export OMERO_DEV_PLUGINS=1
omero metadata populate --report --batch 1000 --file idr0021-experimentA-annotation.csv Project:<project_id>
omero metadata populate --context bulkmap --cfg idr0021-experimentA-bulkmap-config.yml --batch 100 Project:<project_id>
```

I was not part of all the discussion here so I assume there is some rationale to using and maintaining a Python script that achieves the same goal.

Reply:
I didn't know about ```omero metadata populate``` and I already had a similar Python script which only needed a bit of tweaking. But I could give that a try.

Reply:
@sbesson I think that the objective here is quite different from the objectives in IDR. The main objective here is to provide easy-to-follow training material. It is easy to follow that the MapAnnotations are being copied from IDR, because IDR is well-established and maintained source of data. 
Much harder is to follow path of git repo -> csv -> omero metadata populate. Maybe not so much from the point of view of technical feasibility, but from the teaching perspective it is much easier to understand, and goes a bit in the direction of ``have one OMERO server and transfer the data to another OMERO server from it``.  Which is quite easy to understand and what the users want.